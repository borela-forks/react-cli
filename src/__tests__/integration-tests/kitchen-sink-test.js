// @flow strict

import React from "react";
import TestRender from "./test-render";

TestRender(
  "Columns and rows should work together",
  <div>
    Some text
    <div horizontal>
      <div>Column A</div>
      <div>Column B</div>
      <div>Column C</div>
    </div>
    Other text
  </div>
);

TestRender(
  "Columns should work nested within other columns",
  <div horizontal>
    <div>Column A</div>
    <div>
      Column B
      <div horizontal align="center">
        Some other text that should probably wrap
        <br />
        And heres the other column
      </div>
    </div>
    <div>Column C</div>
  </div>
);
