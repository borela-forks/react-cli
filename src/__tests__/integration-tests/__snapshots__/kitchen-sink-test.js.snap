// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Columns and rows should work together 1`] = `
"Some text+++++++++++++++++++++++++++++++++++++++++
Column A++++++++Column B++++++++Column C++++++++++
Other text++++++++++++++++++++++++++++++++++++++++
"
`;

exports[`Columns should work nested within other columns 1`] = `
"Column A++++++++Column B++++++++Column C++++++++++
++++++++++++++++++Some++++And+++++++++++++++++++++
+++++++++++++++++other+++heres++++++++++++++++++++
++++++++++++++++++text++++the+++++++++++++++++++++
++++++++++++++++++that+++other++++++++++++++++++++
+++++++++++++++++should++column+++++++++++++++++++
++++++++++++++++probably++++++++++++++++++++++++++
++++++++++++++++++wrap++++++++++++++++++++++++++++
"
`;

exports[`Columns, rows, section styles should all work together 1`] = `
"##################################################
#++++++++++++++++++++Some App++++++++++++++++++++#
#************************------------------------#
#*++++++âœ”ï¸Ž Step 1+++++++*|Some messages for this|#
#*+++++++â—¯ Step 2+++++++*|app+++++++++++++++++++|#
#*+++++++â—¯ Step 3+++++++*------------------------#
#************************++++++++++++++++++++++++#
#+++++++Some stuff for this app is done! ðŸ¤˜+++++++#
#Heres some more informative stuff about your app#
#++++++++++++++++++++browser+++++++++++++++++++++#
#+++++++++++++++++++++â†™â†— â†–â†˜++++++++++++++++++++++#
#------------------------------------------------#
#|++++++++server++++++++||++++++dev-server++++++|#
#|++(initial response)++||+++++(app bundle)+++++|#
#|++++localhost:3000++++||++++localhost:8080++++|#
#------------------------|websocket server (for+|#
#++++++++++++++++++++++++|+++++++++HMR)+++++++++|#
#++++++++++++++++++++++++|++++localhost:8081++++|#
#++++++++++++++++++++++++------------------------#
##################################################
"
`;
